### PR TITLE
v5.0 Changes

### PR DESCRIPTION
**Pull request by justinjeffery-ipf**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -27,8 +27,8 @@ packages = [
 python = "^3.7.1"
 nautobot-chatops = "^1.1.0"
 netutils = "^1.0.0"
-ipfabric-diagrams = "^1.2.7"
-ipfabric = "^0"
+ipfabric-diagrams = "5.*"
```
Comment (whitej6) [inline pyproject.toml:30]: If updating pyproject.toml please run `poetry lock` to regenerate the lock file.

Comment (justinjeffery-ipf) [reply pyproject.toml:30]: I can't because I can't do a poetry install on any of my Windows machines

Comment (pke11y) [reply pyproject.toml:30]: I'll regenerate the lock file.

**Comment by pke11y** [comment]
@whitej6 great, thanks for the info

**Comment by justinjeffery-ipf** [comment]
We are working on many new updates to 5.0 package.  Will hopefully continue working on this next week.

**Comment by whitej6** [comment]
@justinjeffery-ipf is this ready for final review & merge? If so I will pull down and run a local instance to test.

**Comment by justinjeffery-ipf** [comment]
> @justinjeffery-ipf is this ready for final review & merge? If so I will pull down and run a local instance to test.

Yes however this will only work with IP Fabric version 5.0, all support for 4.4 was dropped.

**Comment by pke11y** [comment]
@justinjeffery-ipf what's the incompatibility with v4 - the API URL? 
I'm okay with moving to only supporting v5 on the next release. We'd just need to test to verify all is working okay.

**Comment by chadell** [comment]
I support a clear deprecation policy, something like we are doing in https://github.com/nautobot/nautobot-plugin-golden-config/blob/develop/docs/admin/admin_install.md#deprecation-policy

**Comment by justinjeffery-ipf** [comment]
Table PR till Friday's meeting?

**Comment by pke11y** [comment]
> Should I be on that call?

@whitej6 will sync with you offline on this.

**Review by whitej6 — APPROVED** [review]

**Pull request merged**